src/Compilers/Core/Portable/Symbols/SymbolEqualityComparer.cs

```diff
@@ -26,6 +26,9 @@ public sealed class SymbolEqualityComparer : IEqualityComparer<ISymbol?>
         // Internal only comparisons:
         internal readonly static SymbolEqualityComparer ConsiderEverything = new SymbolEqualityComparer(TypeCompareKind.ConsiderEverything);
 
+        // Internal only comparisons:
+        internal readonly static SymbolEqualityComparer IgnoreEverything = new SymbolEqualityComparer(TypeCompareKind.AllIgnoreOptions);
```

Comment:
>SymbolEqualityComparer [](start = 33, length = 22)

This comparer is not meant for usage by compilers. I think the change to this file should be reverted.